app/app.go

```diff
@@ -88,6 +88,8 @@ import (
 const (
 	AccountAddressPrefix = "celes"
 	Name                 = "celestia-app"
+	// TokenDenom defines the name of the native staking token.
+	TokenDenom = "celes"
```

Comment:
why is this being added? celes is not a denom on chain, 1,000,000uceles = 1 celes

Reply:
This was added to replace the usage of DisplayDenom in the default genesis afaiu